cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -670,7 +679,17 @@ func unlockIfEncrypted(name string, lockKeysOnFinish bool) (string, error) {
 		return "", fmt.Errorf("cannot lock access to sealed keys: %v", lockErr)
 	}
 
-	return device, nil
+	// return the encrypted device if the device we are maybe unlocked is an
+	// encrypted device
```

Comment:
nitpick: The comment sounds awkward. "have unlocked" ? and somehow change it, to not say "encrypted device" twice?
`// return the encrypted device if we attempted to unlock it` ?

Reply:
this was meant to read as 
```
return the encrypted device if the device we are maybe unlock**ing** is an encrypted device
```

Reply:
I'll change this in the master PR, not here